### ISSUE TITLE
Tag/host filter

### ISSUE THREAD

**Issue by alexanderzobnin** [voidfiles/grafana-datasource-datadog#9]
May be make sense to split filter (Scope field) into _Tag_ and _Host_ fields? What you think?

**Comment by voidfiles** [comment]
If you think that makes sense go for it.

In my experience "host" is really just another tag. I think in most place's datadog treats host as such.

Here you can see in their metric explorere that they lump host, and other tags together.

![hosts](http://g.recordit.co/g3o3UmgkOQ.gif)

**Comment by alexanderzobnin** [comment]
Yes, I understand it. I'm thinking about Grafana experience, this approach allows to use template variables as a host name for quick metric filtering. I should read more about datadog tags.
